web/src/store/index.ts

```diff
@@ -1,13 +1,22 @@
 import { cloneDeep } from 'lodash';
 import Vue from 'vue';
-import Vuex from 'vuex';
+import Vuex, { StoreOptions } from 'vuex';
+import { RootState } from '@/types';
+
+interface RowIndex {
+  row_index: number;
+};
+
+interface ColumnIndex {
+  column_index: number;
+};
 
 import {
   convertCsvToRows, RangeList, mapValidationErrors,
-} from '../utils';
-import analyses from '../components/vis/analyses';
-import { plot_types } from '../utils/constants';
-import { CSVService, ExcelService } from '../common/api.service';
+} from '@/utils';
+import analyses from '@/components/vis/analyses';
+import { plot_types } from '@/utils/constants';
+import { CSVService, ExcelService } from '@/common/api.service';
```

Comment:
What do you gain from using `@`?  The only thing I can tell that changes is:

* My editor can no longer produce type hints because it can't resolve these names; and
* My editor can no longer automatically fix all imports if I move a file.

I'm sure I could figure out configuration , but I wonder why this is better.

Reply:
I think it's better for two reasons:
1. It's faster to type because you don't have to think about which level of the directory hierarchy you're in.
2. It enables moving files to other places more readily, in case of a code reorg.

You would still use relative paths for files that are meant to be bundled together in the same directory, etc. (i.e., ones that would move along with the source file in case of a reorg).

Your editor not supporting this is a big deal. I am surprised though: I didn't invent this `@' thing; it comes with Vue CLI projects out of the box.

If this causes major problems for the dev team we shouldn't use it (although I do really like it).

Reply:
I'll see if I can get my editor to like it.  Shouldn't be too hard.